### PR TITLE
Version 07

### PR DESCRIPTION
**Pull request by mwmiller**
- Add TTL command.
- Fix argument unpacking for keys command
- Increment to 0.07

@kavehmz This subsumes the changes in 0.06.  I think a single release of 0.07 is probably the right way to go, but you tell me!

[Review on Reviewable](https://reviewable.io/reviews/binary-com/perl-cache-redisdb/3)

### CODE REVIEW

File: lib/Cache/RedisDB.pm [hunk 1]
```diff
@@ -133,6 +133,34 @@ sub del {
...
 
+=head2 keys($namespace)
+
+Return a list of all known keys in the provided I<$namespace>.
+
+=cut
+
+sub keys {
+    my ($self, $namespace) = @_;
+    my $prefix = $namespace . '::';
```
Comment (einhverfr) [inline lib/Cache/RedisDB.pm]: Given that this means undef will throw a warning, would it be worth documenting this as being required and croaking if $namespace is undef?  Just a thought.

Comment (mwmiller) [inline lib/Cache/RedisDB.pm]: Hrm.  Let me see how the rest of this module handles this.  It's a good point, but I wanna make sure the response is consistent.

**Comment by mwmiller** [comment]
@einhverfr I've just pushed a new version where all methods use a common key builder (`_cache_key`) it turns any undefined part into an empty string.  This should keep the cache (muddying) behavior as it was before, but eliminate concerns about `undef` concatenation.

I also added the Changes for 0.07, since I seem to have lost them in the mix somewhere.  (I really like using Milla all of a sudden which does some sanity checking on my stuff before letting me release.)

**Comment by mwmiller** [comment]
@einhverfr If you have some time to look at this today, I would greatly appreciate it.  I'd like to be able to update cpanfile and get reasonable results from travis.  With this in, I may replace all instances of YAML::XS::LoadFile in user land code.

**Comment by einhverfr** [comment]
I like the changes  I dont see any problem.

**Comment by mwmiller** [comment]
Sweet.  Thanks for the review, @einhverfr!

**Pull request merged**